### PR TITLE
[AS-909] Use SAM to get/set TOS status [Do Not Merge Until TOS is enabled in SAM]

### PR DESCRIPTION
**Pull request by sagehen03**
I'm pretty sure integration tests are failing because the tests involve registering a new user which won't work with these changes until TOS management is enabled for SAM in dev.  As noted in the title, these changes should not be merged until that happens.  I tested with a local SAM.  Please review my changes to ajax.js in particular as if the TOS was already enabled in dev.

<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

**Review by petesantos — COMMENTED** [review]
I took a look at this PR - and the test failures. I did see a screenshot that showed a TOS error. Let me know if you need any help.

**Comment by sagehen03** [comment]
> I took a look at this PR - and the test failures. I did see a screenshot that showed a TOS error. Let me know if you need any help.

Thanks Pete.  The integration test errors are expected.  The plan is to merge to a feature branch and merge into develop only once we've made the switch to use SAM for TOS stuff.  Can you review the code changes and disregard the test failures for now?

**Review by petesantos — APPROVED** [review]
Based on @sagehen03's comments, the test failures are expected. With that, LGTM

File: src/libs/ajax.js [hunk 1]
```diff
@@ -241,9 +239,8 @@ const User = signal => ({
   },
 
   acceptTos: async () => {
-    await fetchOk(
-      `${getConfig().tosUrlRoot}/user/response`,
-      _.mergeAll([authOpts(), { signal, method: 'POST' }, jsonBody({ ...tosData, accepted: true })])
+    await fetchSam('api/users/v1/tos/accept',
+      _.mergeAll([authOpts(), { signal, method: 'POST' }, jsonBody('app.terra.bio/#terms-of-service')])
```
Comment (zarsky-broad) [inline src/libs/ajax.js:243]: Should the URL come from the current location, since the app is deployed at multiple URLs?

**Comment by kyuksel** [comment]
Hi @sagehen03 - are we going to be able to merge this PR anytime soon? If not, can we convert it into a draft PR so it doesn't show up on daily triage?

**Comment by sagehen03** [comment]
> Hi @sagehen03 - are we going to be able to merge this PR anytime soon? If not, can we convert it into a draft PR so it doesn't show up on daily triage?

Sure thing!

**Pull request closed without merging**